### ISSUE TITLE
Contact support when robots.txt 404s

### ISSUE THREAD

**Issue by jeff-matthews** [magento/devdocs#5237]
# General issue

## Description:

I'd like to make a suggestion for https://devdocs.magento.com/guides/v2.2/cloud/trouble/robots-sitemap.html.

## Possible solutions:

Add a troubleshooting section saying that if your `robots.txt` file goes to 404, you should open a Support Ticket requesting to remove the redirect from `/robots.txt to /media/robots.txt`
https://magento.zendesk.com/agent/tickets/238834

Alternatively, to add that bit to the line that says You do not have to generate a robots.txt because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the url: `<domain.your.project>/robots.txt`
basically, if `<domain.your.project>/robots.txt` gives a 404 error, to open a Support ticket
